tests/test_classes.py

```diff
@@ -0,0 +1,60 @@
+"""
+Testing the functions in the classes.py file
+"""
+import numpy as np
+import pyrft as pr
+def test_Field():

```

Comment:
skip 2 lines before the function + add a oneline docstring

Reply:
Fixed and I have added online docstrings to all the test functions.